src/card.css

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+.card__image-product {
+  height: 134px;
+  width: 160px;
+  padding: 32px 19px 0;
+}
+
+.card__title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
Move to CSS variables and move them to :root selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)